schedule/sle-micro/publiccloud/basic.yml

```diff
@@ -5,4 +5,12 @@ description: >
   Maintainer: <qa-c@suse.de>
 schedule:
   - boot/boot_to_desktop
+  - publiccloud/prepare_instance
```

Comment:
Do we actually use this schedule?

Reply:
yes indeed, I preferred to keep it up to date, even being it not used anymore, in case of next rethinking, being here work in progress. 
I was thinking to remove it later in the project: WDYT?

Reply:
Ok

Reply:
I think we need to delete this file . I don't see chances that we will switch to YAML .

Reply:
There is also `upload_img.yml`, obsoleted by the proper code in `sub load_slem_on_pc_tests`: to delete those `yml` I would then create a sepatated PR.

Reply:
ok